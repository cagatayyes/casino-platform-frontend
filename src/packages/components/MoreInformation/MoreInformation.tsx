import Navigation from '../Navigation/Navigation';
import { MoreInformationItems } from '../Navigation/Navigation.constants';
import { INavigationTriggerProps } from '../Navigation/Navigation.type';


export const MoreInformation = (props: INavigationTriggerProps): JSX.Element => {

	return (
		<div className='ml-[10px]'>
			<Navigation
				options={MoreInformationItems}
				classNames='flex flex-row w-full items-left items-start cursor-pointer p-[6px]'
				handleTabClick={props.handleTabClick}
				activeTab={props.activeTab}
			/>
		</div>
	);
};
