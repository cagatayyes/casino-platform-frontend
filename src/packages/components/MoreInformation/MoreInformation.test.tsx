import { render } from '@testing-library/react';
import { MoreInformation } from './MoreInformation';
import React from 'react';

describe('More Information', () => {
	it('should render More Information details', () => {
		const component = render(
			<MoreInformation
				handleTabClick={jest.fn()}
				maxWidth='640px'
				activeTab='Customer Support'
			/>,
		);
		expect(component).toMatchSnapshot();
	});
});
