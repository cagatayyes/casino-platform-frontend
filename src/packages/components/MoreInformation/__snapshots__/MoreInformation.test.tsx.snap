// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`More Information should render More Information details 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="ml-[10px]"
      >
        <div
          class="flex flex-row w-full items-left items-start cursor-pointer p-[6px] "
        >
          <span
            class="w-[20] h-[20] material-symbols-outlined"
            style="font-size: 20px;"
          >
            shield_person
          </span>
          <span
            class="text-[12px] font-semibold pl-[5px]"
          >
            Safe Gambling
          </span>
          <span
            class="absolute right-[16px] mt-[5px] justify-right material-symbols-outlined"
            style="font-size: 11px;"
          >
            arrow_right_alt
          </span>
        </div>
        <hr
          class="w-full border-b-[1px] broder-b-[#ffffff] opacity-[0.1] m-[5px]"
        />
        <div
          class="flex flex-row w-full items-left items-start cursor-pointer p-[6px] text-[#FFB800] bg-[#070C26] border-t-2 border-t-[#FFB800]"
        >
          <span
            class="w-[20] h-[20] material-symbols-outlined"
            style="font-size: 20px;"
          >
            call_quality
          </span>
          <span
            class="text-[12px] font-semibold pl-[5px]"
          >
            Customer Support
          </span>
          <span
            class="absolute right-[16px] mt-[5px] justify-right material-symbols-outlined"
            style="font-size: 11px;"
          >
            arrow_right_alt
          </span>
        </div>
        <hr
          class="w-full border-b-[1px] broder-b-[#ffffff] opacity-[0.1] m-[5px]"
        />
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="ml-[10px]"
    >
      <div
        class="flex flex-row w-full items-left items-start cursor-pointer p-[6px] "
      >
        <span
          class="w-[20] h-[20] material-symbols-outlined"
          style="font-size: 20px;"
        >
          shield_person
        </span>
        <span
          class="text-[12px] font-semibold pl-[5px]"
        >
          Safe Gambling
        </span>
        <span
          class="absolute right-[16px] mt-[5px] justify-right material-symbols-outlined"
          style="font-size: 11px;"
        >
          arrow_right_alt
        </span>
      </div>
      <hr
        class="w-full border-b-[1px] broder-b-[#ffffff] opacity-[0.1] m-[5px]"
      />
      <div
        class="flex flex-row w-full items-left items-start cursor-pointer p-[6px] text-[#FFB800] bg-[#070C26] border-t-2 border-t-[#FFB800]"
      >
        <span
          class="w-[20] h-[20] material-symbols-outlined"
          style="font-size: 20px;"
        >
          call_quality
        </span>
        <span
          class="text-[12px] font-semibold pl-[5px]"
        >
          Customer Support
        </span>
        <span
          class="absolute right-[16px] mt-[5px] justify-right material-symbols-outlined"
          style="font-size: 11px;"
        >
          arrow_right_alt
        </span>
      </div>
      <hr
        class="w-full border-b-[1px] broder-b-[#ffffff] opacity-[0.1] m-[5px]"
      />
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
